packages/commons/mock-utils/index.ts

```diff
@@ -147,7 +154,20 @@ export class WireMock {
         // Format response body with validation and fallback
         let body = "";
         if (bodyObj !== null && bodyObj !== undefined) {
-            if (typeof bodyObj === "object") {
+            if (isSseResponse && Array.isArray(bodyObj)) {
+                // Format SSE response according to the SSE protocol
+                // Each event should be formatted as: event: <event_name>\ndata: <json_data>\n\n
+                const sseEvents = bodyObj as Array<{ event: string; data: { jsonExample?: unknown } }>;
+                body = sseEvents
+                    .map((sseEvent) => {
+                        const eventName = sseEvent.event || "message";
+                        // Access jsonExample from the data field
+                        const eventData = sseEvent.data?.jsonExample ?? {};
+                        const dataJson = JSON.stringify(eventData);
+                        return `event: ${eventName}\ndata: ${dataJson}\n`;
+                    })
+                    .join("\n");
```

Comment:
Ooh so I guess this results in the double newline delimiter which is what we want

Reply:
the response format will looks like this `event: <name>\ndata: <json>\n\n`